aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -300,38 +764,124 @@ In this section, classes are added for managing movies in a cross-platform [SQLi
+
+The `Movie` class contains:
+
+* The `ID` field is required by the database for the primary key.
+* `[DataType(DataType.Date)]`:  The [DataType](xref:System.ComponentModel.DataAnnotations.DataTypeAttribute) attribute specifies the type of the data (Date). With this attribute:
+
+  * The user is not required to enter time information in the date field.
+  * Only the date is displayed, not time information.
 
-[!INCLUDE [model 2](~/includes/RP/model2.md)]
+[DataAnnotations](/dotnet/api/system.componentmodel.dataannotations) are covered in a later tutorial.
+
+<a name="dc"></a>
+
+### Add NuGet packages and EF tools
+
+[!INCLUDE[](~/includes/add-EF-NuGet-SQLite-CLI.md)]
+
+### Add a database context class
+
+In the RazorPagesMovie project, create a new folder called *Data*. 
+Add the following `RazorPagesMovieContext` class to the *Data* folder:
+
+[!code-csharp[](~/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30/Data/RazorPagesMovieContext.cs)]
+
+The preceding code creates a `DbSet` property for the entity set. In Entity Framework terminology, an entity set typically corresponds to a database table, and an entity corresponds to a row in the table. The code won't compile until dependencies are added in a later step.
+
+<a name="cs"></a>
+
+### Add a database connection string
+
+Add a connection string to the *appsettings.json* file as shown in the following highlighted code:
+
+[!code-json[](~/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie/appsettings_SQLite.json?highlight=8-9)]
+
+### Add required NuGet packages
+
+Run the following .NET Core CLI command to add SQLite and CodeGeneration.Design to the project:
+
+```dotnetcli
+dotnet add package Microsoft.EntityFrameworkCore.SQLite
+dotnet add package Microsoft.VisualStudio.Web.CodeGeneration.Design
+dotnet add package Microsoft.EntityFrameworkCore.Design
+```
+
+The `Microsoft.VisualStudio.Web.CodeGeneration.Design` package is required for scaffolding.
+
+<a name="reg"></a>
+
+### Register the database context
+
+Add the following `using` statements at the top of *Startup.cs*:
+
+```csharp
+using RazorPagesMovie.Models;
+using Microsoft.EntityFrameworkCore;
+```
+
+Register the database context with the [dependency injection](xref:fundamentals/dependency-injection) container in `Startup.ConfigureServices`.
+
+[!code-csharp[](~/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie22/Startup.cs?name=snippet_UseSqlite&highlight=11-12)]
+
+Build the project as a check for errors.
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-* In Solution Explorer, right-click the **RazorPagesMovie** project, and then select **Add** > **New Folder**. Name the folder *Models*.
-* Right-click the *Models* folder, and then select **Add** > **New File**.
+* In Solution Explorer, control-click the **RazorPagesMovie** project, and then select **Add** > **New Folder**. Name the folder *Models*.
```

Comment:
UI element names, such as Visual Studio window names, should be bolded. Italicize project names.

```suggestion
* In **Solution Explorer**, control-click the *RazorPagesMovie* project, and then select **Add** > **New Folder**. Name the folder *Models*.
```